nodejs/kubernetesx/kx.ts

```diff
@@ -317,6 +319,10 @@ export class Service extends k8s.core.v1.Service {
                 spec: spec,
             },
             opts);
+
+        this.ipAddress = this.status.loadBalancer.ingress[0].apply(ingress => {
```

Comment:
Given that this could be a `hostname`, "ipAddress" isn't quite accurate. Any alternate suggestions on nomenclature, or is this close enough?

/cc @joeduffy @lukehoban

Reply:
Let's use the term `endpoint`